config/routes.rb

```diff
@@ -95,6 +95,11 @@
     end
 
     resources :apidocs, only: [:index]
+
+    get 'terms_and_conditions', to: 'terms_and_conditions#index'
+    get 'terms_and_conditions/:name/versions/latest', to: 'terms_and_conditions#latest'
+    get 'terms_and_conditions/:name/versions/latest/user_data', to: 'terms_and_conditions#latest_user_data'
+    post 'terms_and_conditions/:name/versions/latest/user_data', to: 'terms_and_conditions#accept_latest'
```

Comment:
Even parameterized this way I could have written them our as resources, but I actually thought this was a bit more clear because of the levels of nesting.